Assets/Text/JavaScript/NodeScripts/QuadTracer.js

```diff
@@ -24,32 +21,27 @@ class QuadTracer {
             this.initialized = true;
             const scene = this.node.findSceneNode();
             if (scene) {
-                scene.layers.background.addChild(this.graphics);
+                scene.layers.middleground.addChildAt(this.graphics, 6);
```

Comment:
What's that 6 ?

Reply:
Oh I forgot that
That is for control the depth but it doesn't seems to work